package.json

```diff
@@ -54,55 +54,55 @@
     "child_process": false
   },
   "dependencies": {
-    "@docsearch/react": "^3.0.0",
-    "@headlessui/react": "^1.5.0",
-    "@next/mdx": "^12.1.4",
-    "classnames": "^2.3.1",
-    "compare-versions": "^6.0.0-rc.1",
-    "gray-matter": "^4.0.3",
-    "next": "12.1.4",
-    "next-mdx-remote": "^4.1.0",
-    "next-seo": "^5.4.0",
-    "prism-react-renderer": "^1.3.5",
-    "react": "17.0.2",
-    "react-click-away-listener": "^2.1.0",
-    "react-copy-to-clipboard": "^5.1.0",
-    "react-dom": "17.0.2",
-    "react-instantsearch-dom": "^6.23.3",
-    "react-scroll": "^1.8.7",
-    "rehype-autolink-headings": "^6.1.1",
-    "rehype-slug": "^5.0.1",
-    "remark-external-links": "^9.0.1",
-    "remark-gfm": "^3.0.1",
-    "remark-inline-links": "^6.0.1",
-    "sass": "^1.50.0",
-    "unist-util-visit": "^4.1.0"
+    "@docsearch/react": "3.3.2",
+    "@headlessui/react": "1.7.7",
+    "@next/mdx": "13.1.2",
+    "classnames": "2.3.2",
+    "compare-versions": "6.0.0-rc.1",
+    "gray-matter": "4.0.3",
+    "next": "13.1.2",
+    "next-mdx-remote": "4.2.0",
```

Comment:
This particular dependency was initially upgraded to 4.2.1 by ncu, but that wasn't compatible with next 13 so I downgraded it to 4.2.0 . See:
 * https://github.com/hashicorp/next-mdx-remote/issues/307#issuecomment-1376580829